### DISCUSSION TITLE
Problem adding LinearTrend

### DISCUSSION THREAD

**Discussion by AviadAvr** [pastas/pastas#674] [Q&A] [answered]
Hi, and thanks for all your efforts in bringing life into Pastas!

Following the examples provided, I've ran into an issue regarding adding a LinearTrend stressmodel, using the following code:

```
ml = ps.Model(head)

rch = ps.RechargeModel(meteo.precip, meteo.evap, name='recharge', rfunc=ps.Gamma())
ml.add_stressmodel(rch)

#trend = ps.LinearTrend(start=head.index[0], end=head.index[-1], name='trend')
trend = ps.LinearTrend(start="2020-01-01", end="2021-01-01", name='trend')
ml.add_stressmodel(trend)

ml.solve(solver=ps.LeastSquares(), freq='h', noise=False)
```

![image](https://github.com/pastas/pastas/assets/124437932/2dcc8672-7a29-491b-a303-548b0640101c)

Needless to say, the model works fine without adding the LinearTrend (albeit having a poor fit), but breaks whenever I do add the trend.
I'm a bit confused by the error message, as I'm not using the Hantush function, and I don't have any NoneType objects.

* Not sure if this is related, but printing trend.tmin and trend.tmax shows:
`1677-09-21 00:12:43.145224193` and `2262-04-11 23:47:16.854775807`, which don't seem to make much sense, regardless of what I set the 'start' and 'end' parameters to be.

Thanks again,
Avi.

**Comment by OnnoEbbens** [comment, marked as answer]
This is a bug in pastas that should be solved. I will convert this into an issue. The error occurs when creating the fit report. So a quick fix to avoid this error might be to add `report=False` to  `ml.solve`.

**Reply by AviadAvr** [reply]
Yep, that did the trick.
Thanks!

**Reply by OnnoEbbens** [reply]
The issue is fixed in the dev branch an will be available in a new release of pastas (probably version 1.4).

**Comment by dbrakenhoff** [comment]
Hi @AviadAvr, thanks for posting your issue. This is a mistake on my part. I'm about to submit a fix. I see Onno provided a workaround for now, so hopefully that allows you to continue for now. 

You can still print the rest of the fit_report manually using:
```python
print(ml.fit_report(warnings=False))
```
